pyvista/plotting/_property.py

```diff
@@ -230,6 +237,16 @@ def __init__(
         self.line_width = line_width
         if culling is not None:
             self.culling = culling
+        if vtk_version_info < (9, 3) and edge_opacity is not None:  # pragma: no cover
+            from pyvista.core.errors import VTKVersionError
+
+            raise VTKVersionError(
+                "`edge_opacity` cannot be used under VTK v9.3.0. Try installing VTK v9.3.0 or newer."
+            )
+        if edge_opacity is None:
+            edge_opacity = self._theme.edge_opacity
+        if not vtk_version_info < (9, 3):
```

Comment:
I dont' think we need this check here, since it is already done in the setter.  If it is kept, I'd slightly prefer using `if vtk_version_info >= (9, 3):` instead.

Reply:
```suggestion
```